srcs/app/http/MimeTypes.cpp

```diff
@@ -0,0 +1,125 @@
+#include <MimeTypes.hpp>
+#include <utility>
+
+const MimeTypes::MimeTypesMap_
+MimeTypes::kMimeTypes_ = CreateMimeTypesMap_();
+
+const MimeTypes::MimeTypesMap_
+MimeTypes::CreateMimeTypesMap_() {
+	MimeTypesMap_ mt;
+
+	mt.insert(std::make_pair("html", "text/html"));
```

Comment:
I have the same comments here as in the other [Pull Request](https://github.com/42AGV/webserv/pull/46#discussion_r720800901). Would be nice if we could find an easy way to have all this data in a single config file. And, since we will have the same issue here, we could re-use all that code! I am going to open a simple `task` issue to investigate this.

Reply:
Sure, we could look into it :+1: